core/src/toga/command.py

```diff
@@ -93,6 +93,18 @@ def is_child_of(self, parent: Group | None) -> bool:
             return False
         return parent.is_parent_of(self)
 
+    def descendant(self, parent: Group | None) -> Group | None:
```

Comment:
I don't think there's any reason to make this part of the public API.

Also, suggest renaming `parent` to `ancestor`, and removing`None` as a valid argument type.

Reply:
If it's not public (and I agree it's got limited utility), we might as well keep it entirely internal to the iteration process. I'll refactor on that basis.